src/content/supported-icons.tsx

```diff
@@ -1,21 +1,35 @@
 import { IconApi16 } from '@hashicorp/flight-icons/svg-react/api-16'
 import { IconConnection16 } from '@hashicorp/flight-icons/svg-react/connection-16'
+import { IconCloud16 } from '@hashicorp/flight-icons/svg-react/cloud-16'
+import { IconDatabase16 } from '@hashicorp/flight-icons/svg-react/database-16'
 import { IconDocs16 } from '@hashicorp/flight-icons/svg-react/docs-16'
 import { IconDownload16 } from '@hashicorp/flight-icons/svg-react/download-16'
+import { IconEnterprise16 } from '@hashicorp/flight-icons/svg-react/enterprise-16'
+import { IconFileSource16 } from '@hashicorp/flight-icons/svg-react/file-source-16'
 import { IconGuide16 } from '@hashicorp/flight-icons/svg-react/guide-16'
 import { IconHome16 } from '@hashicorp/flight-icons/svg-react/home-16'
 import { IconPlug16 } from '@hashicorp/flight-icons/svg-react/plug-16'
+import { IconPlusCircle16 } from '@hashicorp/flight-icons/svg-react/plus-circle-16'
+import { IconProvider16 } from '@hashicorp/flight-icons/svg-react/provider-16'
 import { IconTerminal16 } from '@hashicorp/flight-icons/svg-react/terminal-16'
 import { IconTerminalScreen16 } from '@hashicorp/flight-icons/svg-react/terminal-screen-16'
+import { IconTools16 } from '@hashicorp/flight-icons/svg-react/tools-16'
 
 export const SUPPORTED_ICONS = {
 	api: <IconApi16 />,
+	cloud: <IconCloud16 />,
```

Comment:
Added support for icons needed to render Terraform docs nav items, as in design specs.